Tests/SwiftDocCUtilitiesTests/SignalTests.swift

```diff
@@ -32,32 +32,34 @@ class SignalTests: XCTestCase {
     /// `signal-test-app` sends a kill signal to itself and uses ``Signal`` to intercept that signal
     /// and set the exit code to 99.
     func testTrappingSignal() throws {
-        let signalTestAppPath = productDirectory.appendingPathComponent("signal-test-app").path
```

Comment:
Can we disable this test only when running with TSan, so that we can still benefit from it in regular testing?

Reply:
Yes, we can check `ProcessInfo.processInfo.environment.keys.contains("TSAN_OPTIONS")`

Reply:
Actually, this key doesn't appear when run from the command line via `swift test --sanitize thread`. I don't think we can conditionally turn this test on/off based on whether TSan is enabled.